packages/modules/mssqlserver/src/mssqlserver-container.ts

```diff
@@ -0,0 +1,123 @@
+import { AbstractStartedContainer, GenericContainer, StartedTestContainer } from "testcontainers";
+
+const MSSQL_PORT = 1433;
+
+type ProductKey = `${string}-${string}-${string}-${string}-${string}`;
```

Comment:
why is this needed?

Reply:
Product key comes from the official documentation here: 

[https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-configure-environment-variables?view=sql-server-ver16#environment-variables](https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-configure-environment-variables?view=sql-server-ver16#environment-variables)

>If specifying a product key, it must be in the form of #####-#####-#####-#####-#####, where '#' is a number or a letter.

However, if the edition should be added in by the user using withEnvironment, I can remove all PID/Edition code.